ts-nestjs-sequelize/README.md

```diff
@@ -0,0 +1,49 @@
+# ts-nest-sequelize
+
+## Description
+The NestJS + Sequelize sample application is a REST API for managing users built using the NestJS web framework and Sequelize ORM.
```

Comment:
Maybe also point to the website docs?

(definitely not a blocker, and I could see an argument for doing it for all sample-apps in one go)